bundles/pascal/misc/example.pas

```diff
@@ -0,0 +1,53 @@
+{ Comments }
```

Comment:
Blast from the past: I had a another look at this while looking at #185, and adjusted the lexer to deal with this in 7111796.